app/assets/stylesheets/components/team_members.scss

```
@@ -10,3 +10,7 @@
     color: $govuk-link-hover-colour;
   }
 }
+
+.list-item-padding{
```

nit: bracket spacing

```suggestion
.list-item-padding {
```

(maybe we need a SCSS linter)